Sources/TuistCloud/Models/CloudOrganization.swift

```diff
@@ -2,6 +2,28 @@ import Foundation
 
 /// Cloud organization
 public struct CloudOrganization: Codable {
+    public enum Plan: Codable, RawRepresentable {
+        case team, none
```

Comment:
Do we need a `none` entry, or can we just use an optional?

Reply:
None is still a valid plan you can have, so I'd lean to keeping it like this.